.gitignore

```diff
@@ -5,4 +5,7 @@ __pycache__/
 config/maps.env
 config/azure-api.env
 config/chart.env
- 
\ No newline at end of file
+config/apis-and-selection.env
+config/vision_api_key.json
```

Comment:
I tried to insert the value of the key in `apis-and-selection.env`, but for some reason the functions from the Google Cloud Vision API only accept an environment variable with the _path_ to the API key file, not the content itself. Should I still add it to the docker-compose file? Since it is already pointed at in the `apis-and-selection.env`.

I added this [environment setup section](https://github.com/Shared-Reality-Lab/IMAGE-server/tree/ocr-cloud-service-selection-feature/preprocessors/ocr#environment-setup) in the README for the OCR preprocessor, should it stay there? I can move it some place else or add more information if needed.